tsl/src/continuous_aggs/invalidation.c

```diff
@@ -777,16 +793,18 @@ cut_cagg_invalidation_and_compute_remainder(const CaggInvalidationState *state,
  * Clear all cagg invalidations that match a refresh window.
  *
  * This function clears all invalidations in the cagg invalidation log that
- * matches a window. Note that the refresh currently doesn't make use of the
- * invalidations to optimize the materialization.
+ * matches a window, and, optionally, adds the invalidation segments covered
+ * by the window to the invalidation store in the passed in state. These
```

Comment:
I don't understand this at all. What is the invalidation store? I know about hypertable invalidation log and cagg (or materialized hypertable) invalidation log. Also how is `optionally` specified, i.e., when is it done and when not?
What does `the passed in state` mean?
Taking in account my current knowledge about invalidation and working with it I am not be able to understand. I guess if somebody will need to deal with it later, will have hard time to understand too.

Reply:
The function has a parameter called "state", which is of type `CaggInvalidationState`. The caller passes in this state as an argument to the function and it contains the invalidation store. The invalidation store is of type `InvalidationStore`, which is defined in this file/header, and explained more extensively in the large comment at the top of this file.

I am happy to make this more clear, but I am not sure how to do it at this point. There is only on "state" passed into this function with said name. There is only one `InvalidationStore` so IMO, the reference is unambiguous.

Reply:
Thank you for explanation. I still don't understand why `optionally`. Do you mean if the invalidation store is not null in `state`? If so, then I suggest to omit `optionally` as it is confusing.
Now I understand what you mean by `in the passed in state` - I think there is some problem with the grammar, which didn't allow me to understand. May be:
```suggestion
 * matches a window, and, optionally, adds the invalidation segments covered
 * by the window to the invalidation store, which is given in the argument state. These
```

Reply:
Removed optionally and clarified the other parts.